.github/release.yml

```diff
@@ -7,32 +7,32 @@ changelog:
       - duplicate
       - wontfix
   categories:
-    - title: 🙌 Notable Changes
+    - title: '🙌 Notable Changes — We would love for you to test them!' 
```

Comment:
I feel this one works nicely in a beta or RC release, so might be worth manually adding.  But for a full release, it maybe gives a wrong impression that we are relying on community testing rather than appreciating it as a bonus.  What do you think?